### DISCUSSION TITLE
Fetch Size value is zero, the whole table is fetched

### DISCUSSION THREAD

**Discussion by baliramaswamy** [dbeaver/dbeaver#11904] [Q&A] [answered]
Hi,

I see DBeaver sends a cancellation request based on the "Resultset Fetch Size", default is 200 on the version I have. If I change it to 100, query cancels at 100. If I change it to zero it does not send a cancellation request. Is that the exepcted behavior of the product?

Thanks
Balaji

**Comment by uslss** [comment, marked as answer]
Hi @baliramaswamy ,
if I catch you right, it is expected behavior. In case fetch size value is set to 0, the whole table should be fetched.
